libs/datamodel/connectors/mongodb-datamodel-connector/src/lib.rs

```diff
@@ -22,6 +22,8 @@ const CAPABILITIES: &[ConnectorCapability] = &[
     ConnectorCapability::ScalarLists,
     ConnectorCapability::InsensitiveFilters,
     ConnectorCapability::CompositeTypes,
+    ConnectorCapability::FullTextIndex,
```

Comment:
I wonder whether we should get away from ultra fine-grained capabilities and use validations on the connectors as we did for the index name validations.

Reply:
Maybe, let's see after Tom's big refactoring.

Reply:
There's no upcoming big refactoring — I agree capabilities won't scale for datamodel validation, connector methods are better. I'm figuring out how they can interact with the parser database at the moment, it's not easy.